crates/utils/src/signing.rs

```diff
@@ -40,36 +39,24 @@ pub fn verify_signed_builder_message<T: Merkleized>(
     Ok(())
 }
 
-pub fn compute_consensus_signing_root<T: Merkleized>(
+pub fn compute_consensus_signing_root<T: HashTreeRoot>(
     data: &mut T,
     slot: Slot,
     genesis_validators_root: &Root,
     context: &Context,
 ) -> Result<Root, Error> {
     let fork = context.fork_for(slot);
     let fork_version = context.fork_version_for(fork);
-    let domain = compute_domain(
-        DomainType::BeaconProposer,
-        Some(fork_version),
-        Some(*genesis_validators_root),
-        context,
-    )?;
+    let domain = compute_domain(DomainType::BeaconProposer, Some(fork_version), Some(*genesis_validators_root), context)?;
```

Comment:
The vast majority of this PR is stuff like this, where you take something that's been spread over multiple lines for legibility and collapse it to one line. What's the reasoning behind this? Does it remove warnings, or is it just your preferred style?